librz/include/rz_type.h

```diff
@@ -388,6 +418,11 @@ RZ_API bool rz_serialize_types_load(RZ_NONNULL Sdb *db, RZ_NONNULL RzTypeDB *typ
 RZ_API void rz_serialize_callables_save(RZ_NONNULL Sdb *db, RZ_NONNULL RzTypeDB *typedb);
 RZ_API bool rz_serialize_callables_load(RZ_NONNULL Sdb *db, RZ_NONNULL RzTypeDB *typedb, RZ_NULLABLE RzSerializeResultInfo *res);
 
+// Constrained Type
+RZ_API const char *rz_type_cond_tostring(RzTypeCond cc);
```

Comment:
Please add if the return is owned or borrowed: https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md#custom-pointer-modifiers